README.md

```diff
@@ -7,3 +7,10 @@
 This repository contains the configuration and auxiliary applications for OpenSearch infrastructure that is needed to run some services:
 - [Assisted Events Scrape](https://github.com/openshift-assisted/assisted-events-scrape/)
 - [Prow Jobs Scraper](https://github.com/openshift-assisted/prow-jobs-scraper)
+
+## Export Kibana dashboards
+
+1. Log in Kibana
+2. Inspect your browser and locate your cookie
+3. Export your cookie `export COOKIE="xxx"`
+4. `make create-backups URL=xxx` where URL is the Kibana URL
```

Comment:
I think the path part would be good to have it here (or anywhere), as it's kibana specific (and not specific for our environment) and it would greatly help usage. It was the base domain that we shouldn't expose IMO.
Sorry for the confusion

Reply:
NEvermind I should've read the script first :sweat: